### PR TITLE
Ot90 634

### PR DESCRIPTION
**Pull request by maxipaci**
### Basic react routing was added to the project

"/" route:
mainRoute

"/nosotros" route:
nosotrosRoute

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
       "react/jsx-props-no-spreading": "off",
       "no-console": "error",
       "semi": "off",
-      "no-param-reassign": "off"
+      "no-param-reassign": "off",
+      "linebreak-style" : 0
     }
 }
```
Comment (dvelezroman) [inline .eslintrc.json]: do a git merge origin/develop

**Review by dvelezroman — CHANGES_REQUESTED** [review]

**Review by benjaminPla — APPROVED** [review]
Nice!

**Review by andresmonsa — APPROVED** [review]
OK! :)

**Review by andresmonsa — APPROVED** [review]
Good Job!

**Review by verteramofederico — APPROVED** [review]

**Review by dvelezroman — APPROVED** [review]

**Pull request merged**